CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Added: `selector-attribute-operator-space-after` rule.
 - Added: `selector-attribute-operator-space-before` rule.
 - Added: `no-extra-semicolons` rule.
+- Added: `selector-max-depth` rule.
```

Comment:
> oh, sorry, we do not seem to sort, it does not matter do not pay attention

I normally alphabetically order the changelog items (for easier reading) just before the release.